R/create.R

```diff
@@ -2,12 +2,17 @@
 #'
 #' @description
 #' These functions create an R project:
-#'   * `create_package()` creates an R package
+#'   * `create_package()` creates an R package.
 #'   * `create_project()` creates a non-package project, i.e. a data analysis
-#'   project
+#'     project.
+#'   * `r lifecycle::badge("experimental")` `create_quarto_project()` creates a
+#'     Quarto project. It is a simplified convenience wrapper around
+#'     `quarto::quarto_create_project()`, which you should call directly for
```

Comment:
I can't actually link to `quarto::quarto_create_project()` because quarto is only in `Suggests`.